racket-custom.el

```diff
@@ -138,6 +138,15 @@ and `racket-repl-documentation' should look for the search page.
   :safe (lambda (val) (or (stringp val) (eq val 'local)))
   :group 'racket)
 
+(defcustom racket--rkt-remote-source-dir
```

Comment:
Until I read your out-of-band comment, it wasn't clear to me (I was dense I guess) that this feature would involve copying the Racket Mode back end Racket code to the remote server. So I think I would probably rewrite the prose for this doc string, to make that clearer for end users. But that's something I'm happy to do myself in a separate commit.

Reply:
You are right